src/toCurrency/toCurrency.ts

```diff
@@ -0,0 +1,46 @@
+export interface ToCurrencyOptions {
+    separator?: string;
+    radix?: string;
+    noCents?: boolean;
+    decimals?: number;
+    currency?: string;
+}
+
+/**
+ * Formats a number or a string number to a currency format
+ * @param {number} value raw number value
+ * @param {ToCurrencyOptions} options You can control the sperator, radix, currency string and showing cents using these options
+ * @returns {string} The formatted currency string
+ */
+export function toCurrency(value: number | string, options: ToCurrencyOptions = {}): string {
+    let amount: string = "0";
+    const format = (val: number): string => {
+        let num: number;
+        let cents: number;
+        if (val - Math.floor(val) === 0) {
+            num = val;
+        } else {
+            num = Math.floor(val);
+            cents = options && options.noCents ? null : Number((val - Math.floor(val)).toFixed(options && options.decimals ? options.decimals : 2).replace("0.", ""));
```

Comment:
this should be the other way around ?
```suggestion
            cents = options && options.noCents ? Number((val - Math.floor(val)).toFixed(options && options.decimals ? options.decimals : 2).replace("0.", "")) : null;
```

Reply:
It would if I rename `noCents` option to `allowCents` :P. this checks `noCents` option is `true` and it will set `cents` to `null`, otherwise, it will calculate the `cents`